### PR TITLE
v2.0.0-beta.3 blog post

### PR DESCRIPTION
**Pull request by krancour**
cc @karenhchu @vdice 

Preview: https://deploy-preview-15--blog-brigade-sh.netlify.app/2021-09-20-brigade-2-reaches-third-beta-release/

### CODE REVIEW

File: content/2021-09-20-brigade-2-reaches-third-beta-release.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+## SDKs
+
+One reason we've been able to build so many integrations so quickly is that we took the time early on to build great SDKs to bind some of our favorite languages to the v2 APIs. We've actually somewhat surprised ourselves with how easy it is to create new integrations. Indeed, in some of the simpler cases, we've gone from prototype to an alpha release in just one day.
+
+If you'd like to build integrations of your own, have a look at:
+
+* [Brigade SDK for Go](https://github.com/brigadecore/brigade/tree/v2/sdk)
+* [Brigade SDK for JavaScript (and TypeScript)](https://github.com/brigadecore/brigade-sdk-for-js)
+
+Our [Brigade SDK for Rust]() remains a work-in-progress -- and an area where we'd gladly accept community assistance from fellow Rustaceans.
```
Comment (vdice) [inline content/2021-09-20-brigade-2-reaches-third-beta-release.md:56]: Missing link here

File: content/2021-09-20-brigade-2-reaches-third-beta-release.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+So, now's a good time to emphasize that although Brigade v2 remains in beta at this time, its APIs are stable and the team is committed to avoiding breaking changes at all cost. A complete documentation refresh is also well underway. In short, if you're one of our valued community members who has understandably taken a step back while the maintainers have executed this complex pivot, now is a great time to get involved again! And we'll be happy to set aside some great swag for anyone who wants to help kick the tires on a beta release or develop a new integration. More important than swag, the team is very interested in diversifying the ranks of maintainers, not only in terms of demographics, but also in terms of what companies are represented. The project's current state presents a prime opportunity to quickly become a maintainer of a CNCF project -- a great thing to have on your CV!
+
+If you'd like to collaborate with us, please do find us on GitHub or on our [Slack channel](https://kubernetes.slack.com/messages/C87MF1RFD).
```
Comment (vdice) [inline content/2021-09-20-brigade-2-reaches-third-beta-release.md:64]: Perhaps `GitHub` can be a link as well? (To https://github.com/brigadecore/brigade)?

**Review by vdice — APPROVED** [review]
Great post!  Lots of good content here.

File: content/2021-09-20-brigade-2-reaches-third-beta-release.md (hunk 1 above)
Comment (krancour) [reply content/2021-09-20-brigade-2-reaches-third-beta-release.md:56]: Great catch.

File: content/2021-09-20-brigade-2-reaches-third-beta-release.md (hunk 2 above)
Comment (krancour) [reply content/2021-09-20-brigade-2-reaches-third-beta-release.md:64]: Yup... will even link straight to the v2 branch.

**Comment by krancour** [comment]
@vdice feedback addressed. Thanks!

**Pull request merged**